common/djangoapps/util/testing.py

```diff
@@ -90,3 +99,200 @@ def reset_tracker(self):
         Reset the mock tracker in order to forget about old events.
         """
         self.mock_tracker.reset_mock()
+
+
+class ContentGroupTestCase(ModuleStoreTestCase):
```

Comment:
Not verbatim, but this code did move here from `lms/djangoapps/django_comment_client/tests/utils.py`